docs/containerd.md

```diff
@@ -0,0 +1,31 @@
+# conrainerd
```

Comment:
containerd

Reply:
Well let's fix this in master, we don't really care in 2.14 😄

Reply:
I'm just backporting master commit, can we just fix the CI and fix typo in another PR ?

Reply:
typo is still in master BTW: https://github.com/kubernetes-sigs/kubespray/blob/master/docs/containerd.md

Reply:
@mattymo ?